10_labmda/10_04_lambda_underneath/10_04_00_lambda_underneath.cpp

```diff
@@ -7,7 +7,7 @@
 #include <iostream>
 #include <vector>
 
-struct A
+struct a
```

Comment:
please give better name for the `struct`. ` a` as the name has very big chance for clashing. Probably `struct empty`